test/acceptance/integration/steps_lib/general.py

```diff
@@ -59,6 +59,18 @@ def a_new_ngsi10_petition_with_the_service_and_the_subservice(step, ngsi_version
         raise ValueError(
             'The version of ngsi api have to be \'9\' or \'10\', not {ngsi_version}'.format(ngsi_version=ngsi_version))
 
+@step('add the following headers to the petition')
```

Comment:
using unicode string is better, see:
https://docs.python.org/2/tutorial/introduction.html#unicode-strings